scripts/us_census/pep/monthly_population_estimate/preprocess.py

```diff
@@ -258,43 +299,51 @@ def _generate_mcf(self, df_cols: list) -> None:
         """
-        mcf_template = ("Node: dcid:{}\n"
-                        "typeOf: dcs:StatisticalVariable\n"
-                        "populationType: dcs:Person{}{}{}\n"
-                        "statType: dcs:measuredValue\n"
-                        "measuredProperty: dcs:count\n")
-        mcf = ""
+
+        mcf_nodes = []
         for col in df_cols:
+            pvs = []
             residence = ""
             status = ""
             armedf = ""
             if col.lower() in ["date", "location"]:
                 continue
             if re.findall('Resident', col):
-                residence = "\nresidentStatus"
-                status = ": dcs:USResident"
-            if re.findall('ArmedForces', col):
-                residence = "\nresidentStatus"
-                if len(status) == 0:
-                    status = ": dcs:InUSArmedForcesOverseas"
+                if re.findall('InUSArmedForcesOverseas', col):
+                    status = "USResident__InUSArmedForcesOverseas"
                 else:
-                    status = ": dcs:USResident__InUSArmedForcesOverseas"
+                    status = "USResident"
+                residence = "residentStatus: dcs:" + \
+                    status
+                pvs.append(residence)
+            elif re.findall('ArmedForces', col):
+                residence = "residentStatus: dcs:" + \
+                         "InUSArmedForcesOverseas"
+                pvs.append(residence)
             if re.findall('Resides', col):
                 if re.findall('Household', col):
-                    residence = "\nresidenceType"
-                    status = ": dcs:Household"
-            if re.findall('NonInstitutionalized', col):
-                residence = "\ninstitutionalization"
-                status = ": dcs:USC_NonInstitutionalized"
+                    residence = "residenceType: dcs:" + \
+                         "Household"
+                    pvs.append(residence)
             if re.findall('Civilian', col):
-                armedf = "\narmedForcesStatus: dcs:Civilian"
+                armedf = "armedForcesStatus: dcs:Civilian"
+                pvs.append(armedf)
+                if re.findall('NonInstitutionalized', col):
+                    residence = "institutionalization: dcs:" + \
+                            "USC_NonInstitutionalized"
+                    pvs.append(residence)
             if re.findall('Count_Person_InUSArmedForcesOverseas', col):
-                armedf = "\narmedForcesStatus: dcs:InArmedForces"
-            mcf = mcf + mcf_template.format(col, residence, status,
-                                            armedf) + "\n"
+                armedf = "armedForcesStatus: dcs:InArmedForces"
+                pvs.append(armedf)
+            node = _MCF_TEMPLATE.format(dcid=col, xtra_pvs='\n'.join(pvs))
+            mcf_nodes.append(node)
+
+            mcf = '\n'.join(mcf_nodes)
+            #mcf = mcf + _MCF_TEMPLATE.format(col, residence, status,

```

Comment:
why is this commented out?

Reply:
we have added             
node = _MCF_TEMPLATE.format(dcid=col, xtra_pvs='\n'.join(pvs)), so commented out old method. 
Will remove it now